diff --git a/dist/src/index.js b/dist/src/index.js
index 1efd1f365acd4e61e506459041caa9a7321cafcc..5741eb48aec77aeb2ff362674cde201787b8167d 100644
--- a/dist/src/index.js
+++ b/dist/src/index.js
@@ -91170,6 +91170,13 @@ function handleAttribute(str, attr, parent, preserveCase, svelte5Plus, element)
                 }
                 value.push('})');
             }
+            else if (attr.name.startsWith('mochi:')) {
+                name.unshift('...__sveltets_2_empty({');
+                if (!value) {
+                    value = ['__sveltets_2_any()'];
+                }
+                value.push('})');
+            }
             element.addProp(name, value);
         };
     const transformAttributeCase = (name) => {
